src/services/url.ts

```diff
@@ -22,9 +22,22 @@ class UrlService extends BaseService<UrlService.Options> {
   }
 
   init() {
+    this.beautifier = new UrlBeautifier(this.getRoutes(), this.opts.beautifier, this.app.config);
     this.readInitialUrl();
   }
 
+  getRoutes() {
```

Comment:
this method should build the new routes based off of the routes that are passed in (`this.opts.routes`) not the `Routes` global from `@storefront/core`